README.md

```diff
@@ -30,17 +30,15 @@ app.
 
```

Comment:
Should we add a note to "option 1" mentioning these UDFs should be installed as well? (Just a link to the github repo is likely enough)